python/pysixtracklib/beam_elements.py

```diff
@@ -288,9 +288,15 @@ def get(self, objid):
         return self.cbuffer.get_object(objid)
 
     @classmethod
-    def from_mad(cls, seq):
+    def from_mad(cls, seq, drift_exact=True):
+        if drift_exact:
+            drift = Elements.element_types['Drift']
+            Elements.element_types['Drift'] = DriftExact
         line = madseq_to_line(seq)
-        return cls.fromline(line)
+        instance = cls.fromline(line)
```

Comment:
I prefer to implement the option `exact` in the `fromline` to avoid using side-effect  base implementation. e.g.

```
    @classmethod
    def fromline(cls, line,exact=False):
        self = cls()
        for label, element_name, element in line:
             if element_name == 'Drift' and exact is True:
                  element_name='DriftExact'
             getattr(self, element_name)(**element._asdict())
        return self

```

Reply:
Sure, I agree. Does this look better? I still opt for "exact_drift" as just "exact" is perhaps not exactly clear.. ;)